### ISSUE TITLE
About the training of discriminator

### ISSUE THREAD

**Issue by xiumin123** [otenim/GLCIC-PyTorch#26]
This happened when I was training, the loss of Phase2 was zero, but in the third stage,loss(cd)=0.00030,loss(cn)=0.00090.What is the problem？
